client/src/ui/product/CommandPalette.vue

```diff
@@ -8,24 +8,44 @@
 
   const showDialog = ref(false);
 
-  const { nameToBindingKeys } = graph.value.shortcut;
+  const { platformBindings } = graph.value.shortcut;
+
+  const bindings = computed(() => {
+    const result: Record<string, string> = {};
+    const bindingNames = Object.entries(platformBindings);
+    for (const [name, bindingInfo] of bindingNames) {
+      result[name] = bindingInfo.binding;
+    }
+    return result;
+  });
+
+  const keyToNameMap: Record<string, string> = {
+    meta: 'Command',
+    ctrl: 'Control',
+  };
+
+  const keyToIconMap: Record<string, string> = {
```

Comment:
same thing as above for this. We should keep in mind key should be keyboard key and call it something like KEYBOARD_KEY_TO_ICON

and `as const`